src/gameplay/cards/MonsterCard.js

```diff
@@ -77,23 +77,54 @@ class MonsterCard extends Card {
   }
 
   takeSnapshot() {
+    let summonAction;
+
+    if (this.lv < 5) {
+      summonAction = {
```

Comment:
Create a new method to construct summonAction object given parameters like 'name' and 'desc'. Since summonAction objects seems to created often in this PR, this refactoring will remove a lot of redundant code when creating a new summonAction object.

Reply:
Done.

Created `createAction` and `createActionParam` functions to create actions and its params.